### ISSUE TITLE
Disable Password Login Field

### ISSUE THREAD

**Issue by syleishere** [NearHuscarl/flutter_login#159]
I did my fastapi so that no passwords are needed anymore using oAuth2 tokens and sending link to their email address, any way to disable this password field?

**Comment by surister** [comment]
I don't believe there is currently a way to disable the password field as per `FlutterLogin` only allows for
 ```
bool hideForgotPasswordButton = false
bool hideSignUpButton = false
```
You could just skip the flutter_login part.

**Comment by juliansteenbakker** [comment]
I don't think there is any use for flutter_login if you don't need a password. I'm closing this for now. Please feel free to comment if you think this should be different.

**Comment by syleishere** [comment]
You don't want to use passwords, or second you start into android tv programming, your gonna wish you didn't in first place from experience trying to support many devices and OS's at once

**Comment by juliansteenbakker** [comment]
> You don't want to use passwords, or second you start into android tv programming, your gonna wish you didn't in first place from experience trying to support many devices and OS's at once

I can understand the use case for when a password just isn't user friendly to use, however i'm not quite sure if we should implement such a flow in this package

**Comment by surister** [comment]
I guess that there could be situations where perhaps you want to login with just an ID or CODE thus needing only one field. 
And you want to use FlutterLogin to use the fancy animations and not just a boring text field.

But there are probably other bigger issues / features that are more important or relevant to the current package usage.

**Comment by syleishere** [comment]
Mainly what you want to do is store a .json file of username and oauth2 password and just log them in, if does not exist, then redirect to fastapi to send them an email to click on. That link should populate their .json file afterwards. Most cross compatible way of doing it I have found.

So say your logging someone into a TV, all they have is a remote, they quickly grab their phone, check their email and boom they are done type thing.

Dan.
________________________________

**Comment by eggzotic** [comment]
I have a login flow where the login-passwd is a timed-OTP (e.g. a code from Google Authenticator), followed by an email-sent-code. Flutter login works well for the sign-in flow (using 2 separate FlutterLogins with slightly differing config). 

*However*, for the sign-up flow, we want the user's email-addr and first & last-names (for which I assume I can use the additional-signup-data fields) - but no user-entered-passwd. So being able to hide the password/confirm-password fields when in sign-in mode is required. I guess this still is not possible?
